ui/src/translations/en.json

```diff
@@ -84,13 +84,26 @@
   "no": "No",
   "ok": "OK",
   "volume_creation": "Volume creation",
-  "volume_creation_success": "Volume {name} has been created successfully.",
+  "volume_creation_success": "Volume {name} is being created.",
```

Comment:
You also need to fix the French translation: `Volume {name} est en cours de création.`